rllib/core/rl_module/rl_module.py

```diff
@@ -357,11 +357,11 @@ class RLModule(abc.ABC):
         config: The config for the RLModule.
 
     Abstract Methods:
-        :py:meth:`~_forward_train`: Forward pass during training.
+        ``~_forward_train``: Forward pass during training.
```

Comment:
Dumb question: In general, not just here, should we or not use `:py:meth:`, `:py:class:`, etc..?

Reply:
If you want to have a link, use `:meth:` or `:py:meth:` or similar. If you want to have monospaced code use single or double backticks.

Here we remove the `:meth:` because `_forward_train` is a private method and thus not documented by default by `autosummary` [here in the RLModule API reference](https://github.com/ray-project/ray/blob/32956415cc5a4402fd7f0c3d25385001908fec44/doc/source/rllib/package_ref/rl_modules.rst?plain=1#L57), which means that any _reference_ to this method then fails. If you want this to be a link, we can explicitly tell `autosummary` to document certain methods specifically, or _all_ methods including private methods; if we do either of those two options then we can keep this as a reference, and the resulting documentation will include a link to the documentation for this method.